service/src/test/java/com/commercetools/MainTestRunner.java

```diff
@@ -0,0 +1,45 @@
+package com.commercetools;
+
+import com.commercetools.pspadapter.payone.IntegrationService;
+import com.commercetools.pspadapter.payone.ServiceFactory;
+import com.commercetools.pspadapter.payone.config.PropertyProvider;
+import com.commercetools.pspadapter.payone.config.ServiceConfig;
+
+import java.net.URISyntaxException;
+
+import static com.commercetools.Main.bridgeJULToSLF4J;
+import static com.commercetools.Main.configureAccessLogs;
+import static com.commercetools.Main.configureLogLevel;
+
+public class MainTestRunner {
```

Comment:
This is not a proper integration test we expected. The class Main.java is not tested and I find it important to test it. I would recommend to revert the easiest test of e2e, because there we tested everyting. Or can you just run Main.java:main() method from the test?

Reply:
6f9f1573f0f12e894483bea2880f3476c0780601